### ISSUE TITLE
Local preview is giving me a list pointing to posts/ and the online version is different

### ISSUE THREAD

**Issue by njtierney** [rbind/njtierney.com#19]
Currently I'm in this weird spot where I get this online:

<img width="1049" alt="image" src="https://user-images.githubusercontent.com/6488485/66101386-0168c300-e5f2-11e9-9f43-df7862d87912.png">

But I get this locally: 

<img width="1048" alt="image" src="https://user-images.githubusercontent.com/6488485/66101391-09c0fe00-e5f2-11e9-871b-d1325a060421.png">

It seems like I have a different thing set up for local vs online setup.

I would have thought that could be due to not having the same verison of hugo online and locally, but I have a [netlify.toml](https://github.com/rbind/njtierney.com/blob/master/netlify.toml), 

And it is being detected on the netlify builds:

```
...
3:23:53 PM: Finished fetching cache in 8.485165229s
3:23:53 PM: Starting to prepare the repo for build
3:23:53 PM: Preparing Git Reference refs/heads/master
3:23:54 PM: Found netlify.toml. Overriding site configuration
...
```

Feeling a bit stuck here - throwing out a(nother) lifeline to @apreshill

**Comment by njtierney** [comment]
OK so now I get the same problem online at njtierney.com - so at least it is consistent?

**Comment by njtierney** [comment]
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/6488485/66102022-3ece5000-e5f4-11e9-8eb4-bc763e6f0dbc.png">

**Comment by njtierney** [comment]
resolved in #20
